app/config/app-template.js

```diff
@@ -1,10 +1,9 @@
-import { HOST, PORT, ENV } from './env';
+import { HOST, PORT, ENV } from '../../env';
 
 // Wordpress
 export const SITE_NAME = 'Starward';
-export const WP_URL = 'http://localhost/starward_wp';
-export const WP_API = `${WP_URL}/wp-json`;
-export const WP_AUTH = new Buffer('username:password').toString('base64');
+export const WP_URL = 'http://localhost/starward_wp'; // put on server side once apollo done
```

Comment:
originally this comment was added as I was still using the wp url for the form, I no longer am, however a utility function for the Navigation that finds out if this is the active link. We need to fix this in issue #76